glide-core/redis-rs/redis/src/aio/connection.rs

```diff
@@ -403,7 +402,7 @@ fn log_conn_creation<T>(conn_type: &str, node: T, ip: Option<IpAddr>)
 where
     T: std::fmt::Debug,
 {
-    info!(
+    tracing::debug!(
```

Comment:
This log can help us later on to debug customer issues (it allows to cross IP with DNS), and it shouldn't be done too often